### DISCUSSION TITLE
Github isn't a social media site and shouldn't take steps towards being one.

### DISCUSSION THREAD

**Discussion by CKolkey** [community/community#13457] [Other Feature Feedback, Questions, & Ideas]
Its alarming to me that a work tool would integrate a feature like this.

**Comment by dragonejt** [comment]
I disagree, to me GitHub is an open source community and should have some social aspects. Perhaps this shouldn't be integrated into GitHub Enterprise, but the best thing about GitHub is its open source community, and the social aspects are beneficial towards that.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
